highfive/newpr.py

```diff
@@ -105,16 +105,25 @@ def api_req(self, method, url, data=None, media_type=None):
 
     def set_assignee(self, assignee, owner, repo, issue, user, author, to_mention):
         try:
+            assignees = [] if assignee == 'ghost' else [assignee]
+
             self.api_req(
                 "PATCH", issue_url % (owner, repo, issue),
-                {"assignee": assignee}
+                {"assignees": assignees}
```

Comment:
This PR also updates highfive to no longer use `assignee`: it's deprecated, and I don't we would be able to clear assignees easily otherwise. I'm not sure what the old behavior was when there were multiple assignees though. When looking at the paylod, I just look at the first assignee.

Reply:
Actually now I'm just checking if the assignee is a member of `assignees`.